### PR TITLE
Add Behringer-CMD-PL-1 controller

### PR DESCRIPTION
**Pull request by yledoare**
Rebased from 2.2

### CODE REVIEW

**Comment by Holzhaus** [comment]
Thank you, pre-commit issues have been fixed. Note that you could just have force-pushed to the same branch instead of opening a new PR, but that doesn't matter now.

Starting with Mixxx 2.3, all controller mappings are now supposed to be documented in the Manual. This is necessary so that other users know how to use this mapping. We also need it to review it properly and check if the mapping makes sense and if the code matches the documentation.

Please fork the [mixxxdj/manual](https://github.com/mixxxdj/manual) repository, check out the `manual-2.3.x` branch and add documentation to the [`source/hardware/controllers` directory](https://github.com/mixxxdj/manual/tree/manual-2.3.x/source/hardware/controllers). Open the pull request and add the link to the PR description.

Here's an example how such documentation could look like: https://manual.mixxx.org/2.3/en/hardware/controllers/roland_dj_505.html (If you don't have schematic images and don't want to draw them yourself, just use the same numbering as in the controller's manual and document the buttons like that).

File: res/controllers/Behringer CMD PL-1.midi.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<?xml version='1.0' encoding='utf-8'?>
+<MixxxControllerPreset mixxxVersion="2.4.0+" schemaVersion="1">
```
Comment (Holzhaus) [inline res/controllers/Behringer CMD PL-1.midi.xml:2]:
```suggestion
<MixxxControllerPreset mixxxVersion="2.2.0+" schemaVersion="1">
```

**Comment by Holzhaus** [comment]
I'll convert this to draft until we have a pending manual PR.

**Comment by uklotzde** [comment]
Please rebase on 2.3 and then force push.

I will change the milestone to 2.3.1 because I don't expect that this PR will be ready for 2.3.0.

**Comment by Holzhaus** [comment]
Why did you merge main? This makes it impossible to merge into 2.3. Please rebase on 2.3. and remove all commit from main.

**Comment by yledoare** [comment]
> Why did you merge main? This makes it impossible to merge into 2.3. Please rebase on 2.3. and remove all commits from main.

This was only for testing.. sorry. Rebase like this :  "git rebase --onto upstream/2.3 upstream/pr/3246 Behringer-CMD-PL-1" ?

**Comment by yledoare** [comment]
Maybe the simpler for me should be creating a new PR

**Comment by Holzhaus** [comment]
> Maybe the simpler for me should be creating a new PR

No, this makes no difference because you still need to rebase, but makes review harder because discussion is split over multiple PRs.

**Comment by uklotzde** [comment]
Any progress on this? Rescheduling to 2.3.2.

**Comment by yledoare** [comment]
Hello,
I have only two files to add, for this controller working with Mixxx, please give me some instruction to add them by the simplest way

Regards.

**Comment by git-developer** [comment]
Looks like this PR is obsolete for follow-up #4493 (targeting 2.3). Is this correct, @yledoare?

**Comment by ronso0** [comment]
superseded by #11518

**Pull request closed without merging**